plugins/optionrom/README.md

```diff
@@ -1,5 +1,5 @@
-UDev Support
-============
+OptionROM Support
+=================
 
 Introduction
 ------------
```

Comment:
Line 21 of this file probably needs updating too

Reply:
It's still valid, no? It's just doing the additional instance ID in the device, not in the FuRom helper.

Reply:
I guess.